src/pages/signin/SignInHeroImage.js

```diff
@@ -1,30 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import Lottie from '@components/Lottie';
 import LottieAnimations from '@components/LottieAnimations';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import variables from '@styles/variables';
 
-const propTypes = {
-    ...windowDimensionsPropTypes,
-
-    shouldShowSmallScreen: PropTypes.bool,
-};
-
-const defaultProps = {
-    shouldShowSmallScreen: false,
-};
-
-function SignInHeroImage(props) {
+function SignInHeroImage() {
     const styles = useThemeStyles();
+    const {isMediumScreenWidth} = useWindowDimensions();
```

Comment:
we should also return `isMediumScreenWidth` from `useResponsive` layout, to avoid the lint-rule we add for `useWindowDimensions` usage?